games-misc/parsec-bin/parsec-bin-147.ebuild

```diff
@@ -0,0 +1,58 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit gnome2-utils unpacker
+
+PARSEC_P="parsec-bin-${PV}"
+
+DESCRIPTION="Simple, Low-Latency Game Streaming - Play, watch, share games with your friends"
+HOMEPAGE="https://parsecgaming.com/"
+SRC_URI="https://s3.amazonaws.com/parsec-build/package/parsec-linux.deb -> $PARSEC_P.deb"
+
+LICENSE="all-rights-reserved"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE=""
+
+RDEPEND="
+	media-libs/freetype
+	media-libs/libao
+	media-libs/mesa
+	x11-libs/cairo
+	x11-libs/gdk-pixbuf
+	x11-libs/gtk+
+	x11-libs/libSM
+	x11-libs/libX11
+	x11-libs/libXxf86vm
+	x11-libs/pango
+	x11-terms/xterm
+"
+
+S="${WORKDIR}"
+
+RESTRICT="mirror bindist"
```

Comment:
Well, this is going to be problematic since your SRC_URI doesnt point to an exact release. What will happen when upstream releases, let's say, 151? Does their license prevent Gentoo from mirroring it?

Reply:
https://gitweb.gentoo.org/repo/gentoo.git/plain/licenses/all-rights-reserved Looks like thats it.

Reply:
In fact, the application is auto-updated (when an update is available) when you launch it.

When you launch the application for the 1st time, a folder is create in ~/.parsec with these file "parsecd-147-1.so". Like i said, if an update is available in the parsec server, it's automatically updated. In my case, it's currently "parsecd-149-2.so".